luda-editor/luda-editor-client/src/app/editor/sequence_player/mod.rs

```diff
@@ -0,0 +1,230 @@
+        if let Some(event) = event.downcast_ref::<SequencePlayerEvent>() {
+            match event {
+                SequencePlayerEvent::CheckLoading(id) => {
+                    if id.ne(&self.id) {
+                        return;
+                    }
+                    match self.content_loader.is_loaded() {
+                        false => self.call_loading_timeout(),
+                        true => {
+                            if !self.is_paused {
+                                self.start_play()
+                            }
+                        }
+                    }
+                }
+                SequencePlayerEvent::AnimationFrame(id) => {
+                    if id.ne(&self.id) || self.is_paused {
+                        return;
+                    }
+                    let id = id.clone();
+                    namui::request_animation_frame(move || {
+                        namui::event::send(SequencePlayerEvent::AnimationFrame(id))
+                    });
+                }
+            }
+        } else if let Some(event) = event.downcast_ref::<ButtonsEvent>() {
+            match event {
+                ButtonsEvent::PlayButtonClicked => self.play(),
+                ButtonsEvent::PauseButtonClicked => self.pause(),
+            }
+        }
+    }
+    fn start_play(&mut self) {
+        if self.is_paused || self.started_at.is_some() || !self.content_loader.is_loaded() {
+            return;
+        }
+
+        self.started_at = Some(Time::now());
+        let id = self.id.clone();
+        namui::request_animation_frame(|| {
+            namui::event::send(SequencePlayerEvent::AnimationFrame(id))
+        });
+    }
+    pub fn render(&self, props: &SequencePlayerProps) -> RenderingTree {
+        let wh = props.xywh.wh();
+        // NOTE : will be translated by props.xywh.xy.
+
+        let title_header_center_y = wh.height * (0.5 / 6.0);
+        let title_header_center_x = wh.width * 0.5;
+        let title_font_size = title_header_center_y.floor() as i16;
+
+        let player_screen_xywh = namui::XywhRect {
+            x: 0.0,
+            y: wh.height * (1.0 / 6.0),
+            width: wh.width,
+            height: wh.height * (4.0 / 6.0),
+        };
+
+        let buttons_xywh = namui::XywhRect {
+            x: 0.0,
+            y: wh.height * (5.0 / 6.0),
+            width: wh.width,
+            height: wh.height * (1.0 / 6.0),
+        };
+
+        let border = namui::rect(namui::RectParam {
+            x: 0.0,
+            y: 0.0,
+            width: wh.width,
+            height: wh.height,
+            style: namui::RectStyle {
+                stroke: Some(namui::RectStroke {
+                    color: Color::BLACK,
+                    border_position: namui::BorderPosition::Inside,
+                    width: 1.0,
+                }),
+                ..Default::default()
+            },
+        });
+        let title_header = namui::text(namui::TextParam {
```

Comment:
I think we don't need title header. do you think so?

Reply:
I thnk so.
In near future, sequence title will be displayed on top bar. maybe...